File: cfgs/pythoncfgs/CHECK_IF_ALL_THE_ELEMENTS_CAN_BE_MADE_OF_SAME_PARITY_BY_INVERTING_ADJACENT_ELEMENTS.py
Nodes
1;statement;foo_1 = 0;8;
2;statement;foo_1 = 0;8;
3;conditional;fun_1(foo_1);9;
4;conditional;foo_1 % 2 and foo_2 % 2;14;
5;statement;return true;17;
6;statement;return false;15;
7;conditional;foo_1[foo_2] & 1;10;
8;statement;foo_1 += 1;13;
9;statement;foo_1 += 1;11;
10;exit;;
Edges
0;2;
1;3;
2;4;7;
3;5;6;
4;10;
5;10;
6;8;9;
7;3;
8;3;
9;

File: cfgs/java_cfgs_foo/CHECK_IF_ALL_THE_ELEMENTS_CAN_BE_MADE_OF_SAME_PARITY_BY_INVERTING_ADJACENT_ELEMENTS.java
Nodes
1;special;;;
2;statement;int count_odd = 0;14;
3;statement;int count_even = 0;14;
4;statement;int i = 0;15;
5;conditional;foo_1 < foo_2;16;
6;conditional;(foo_1[foo_2] & 1) == 1;18;
7;statement;count_odd++;18;
8;statement;count_even++;19;
9;statement;i++;17;
10;conditional;((foo_1 % 2) == 1) && ((foo_2 % 2) == 1);21;
11;statement;return false;21;
12;exit;;;
13;statement;return true;22;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;10;
5;7;8;
6;9;
7;9;
8;5;
9;11;13;
10;12;
11;
12;12;
